static/js/dismiss_banner.js

```diff
@@ -20,22 +20,54 @@ $(document).ready(function() {
   }
 
   /* Check the presence of a cookie */
-  let announcement = document.querySelector("#announcement");
+  /*let announcement = document.querySelector("#announcement");
   let token = `announcement_ack_${announcement.getAttribute('data-announcement-name').replace(/\s/g, '_')}`; // Generate the unique token for announcement
   let acknowledged = getCookie(token);
   if (acknowledged === "true") {
     announcement.remove(); // Remove the announcement if the cookie is set
   }
   else {
     announcement.classList.add('display-announcement') // Display the announcement if the cookie is not set
+  }*/

```

Comment:
```suggestion
```
Let's drop the comment here.

Reply:
Hi @sftim,

Thank you so much for your suggestion. I have updated the PR to remove the comment and make the code cleaner.

Please let me know if there is anything else you would like me to fix.